pytypes/typechecker.pyi

```diff
@@ -0,0 +1,48 @@
+# Generated using mypy stubgen, then editing to add types for some definitions.
+
+import __builtin__ as builtins
+from . import type_util, util
+from .exceptions import InputTypeError, OverrideError, ReturnTypeError
+from .type_util import TypeAgent, deep_type, get_orig_class, has_type_hints, is_Tuple, is_builtin_type, type_str
+from .util import getargspecs
+from types import ModuleType
+from typing import Any, Optional, Callable, TypeVar, Union
+from warnings import warn
+
+_python___import__ = builtins.__import__
```

Comment:
This line should not be required in the stub file, should it?

Reply:
Removed.